From 800be61c9503d7446d9c59f744322bcb45eb4b62 Mon Sep 17 00:00:00 2001
From: yufangxian <yufangxian@eswincomputing.com>
Date: Wed, 14 May 2025 18:21:01 +0800
Subject: [PATCH 367/540] fix: fix dc endpoint index

Changelogs:
1. fix dc endpoint index changed by last vi merge

Signed-off-by: yufangxian <yufangxian@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts b/arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts
index 8d2ad1b263c77..2b22b12ab3894 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts
+++ b/arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts
@@ -308,12 +308,12 @@ &dc {
 	dc_out: port {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		dc_out_dpi: endpoint@0 {
-			reg = <0>;
+		dc_out_dpi: endpoint@1 {
+			reg = <1>;
 			remote-endpoint = <&vd_input>;
 		};
-		dc_out_hdmi: endpoint@1 {
-			reg = <1>;
+		dc_out_hdmi: endpoint@2 {
+			reg = <2>;
 			remote-endpoint = <&hdmi_in_dc>;
 		};
 	};
-- 
2.53.0

